tests/unit/amo/components/TestLandingPage.js

```diff
@@ -137,26 +164,33 @@ describe('<LandingPage />', () => {
         result: { count: 50, results: ['pop', 'pop-again'] },
       },
     }));
-    const root = render({
+    const rootNode = renderNode({
       ...mapStateToProps(store.getState()),
       params: { visibleAddonType: visibleAddonType(ADDON_TYPE_THEME) },
     });
 
-    expect(Object.values(root.querySelectorAll('.SearchResult-heading'))
+    expect(Object.values(rootNode.querySelectorAll('.SearchResult-heading'))
       .map((heading) => heading.textContent)).toEqual(['Howdy', 'Howdy again', 'High', 'High again', 'Pop', 'Pop again']);
   });
 
   it('renders not found if add-on type is not supported', () => {
-    const root = render({ params: { visibleAddonType: 'XUL' } });
-    expect(root.textContent).toContain('Page not found');
+    const rootNode = renderNode({ params: { visibleAddonType: 'XUL' } });
+    expect(rootNode.textContent).toContain('Page not found');
   });
 
   it('throws for any error other than an unknown addonType', () => {
     expect(() => {
       render({
         apiAddonType: () => { throw new Error('Ice cream'); },
-        params: { visibleAddonType: 'doesnotmatter' },
+        params: { visibleAddonType: 'does not matter' },
```

Comment:
Actually this code is here to protect against 404s that hit this page because of the lack of regexes for routes in the router. Anything at the top level gets sent to this component right now so if it isn't a `validAddonType` we 404. I guess really this value would be a valid addonType but something else would cause the error so I'll change this to `ADDON_TYPE_EXTENSION`. Thanks.